### PR TITLE
London class7 ahmad janah

### PR DESCRIPTION
**Pull request by AhmadJanah**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by MSUMBA-art** [comment]
nice work buddy, congraturations.

what about in mandatory _3-playing-computer.js_  if you can delete all th b and the last console.log(b) , then run the code and see what will happen

**Comment by AhmadJanah** [comment]
> nice work buddy, congraturations.
> 
> what about in mandatory _3-playing-computer.js_ if you can delete all th b and the last console.log(b) , then run the code and see what will happen

Hi buddy,

Thanks for your comments,

Could you please explain more as I can't find any "console.log(b)" in my code and they asked us to predict without running the code!

Regards,

**Comment by MSUMBA-art** [comment]
The easiest way is to replace all B, with X

And delete the last concole.log(b).

Just delete all B and put X, wherever there's b

File: exercises/C-comparison-operators/exercise.js [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = (studentCount+mentorCount) <= roomMaxCapacity; // finish this statement
 
 var personA = "Daniel";
 var personB = "Irina";
-var sameName; // finish this statement
+var sameName = personA == personB; // finish this statement
```
Comment (bisioolajide) [inline exercises/C-comparison-operators/exercise.js:17]: Great work Ahmad, there's just a minor point that I noticed, where you used "==", I believe we are advised to always use "===".

**Pull request closed without merging**